pubsub/tests/unit/pubsub_v1/subscriber/test_subscriber_client.py

```diff
@@ -38,19 +40,26 @@ def test_init_emulator(monkeypatch):
     channel = client.api.subscriber_stub.Pull._channel
     assert channel.target().decode('utf8') == '/baz/bacon/'
 
+class TestSubscribe(unittest.TestCase):
+    def test_subscribe(self):
```

Comment:
Please remove this class. This module was written in the `pytest` style and is fine in it's current format.

Also, if you **were** to leave it as a class, put an empty line between the class declaration and the first method. (But just delete the class, this is for next time.)

Reply:
So do I import pytest for pytest.raises?

Reply:
Yes.